Service-Broker-Setup.md

```diff
@@ -0,0 +1,186 @@
+# Setting up and using a service broke sidecar
+
+We currently provide helm charts for two service brokers managing
+access to `MySQL` and `Postgres` databases.
+
+This document describes how to use these charts in the context of
+a CAP cluster.
+
+While the document's examples focus on deploying a `MySQL` database
+the steps are virtually identical for `Postgres`. The
+[Postgres](#appendix-i-postgres) appendix describes the
+differences.
+
+The second appendix lists all relevant chart variables and their
+meanings.
+
+# Assumptions of this document
+
+## About the chart to be deployed
+
+* The user knows how to get the service broker charts, or has them
+  already. This also implies that the user knows the location of the
+  charts in the filesystem.
+
+* The user has stored the path to the chart in the environment
+  variable `CHART`.
+
+  ```
+  CHART=/path/to/mysql-chart
+  ```
+
+* The user has chosen the kube namespace for the broker to run in and
+  stored it in the environment variable `NAMESPACE`.
+
+  ```
+  NAMESPACE=mysql
+  ```
+
+* The user knows docker repository and organization in that repository
+  for the docker images needed by the chart(s) and stored them in the
+  environment variables `DOCKER_ORGANIZATION` and `DOCKER_REPOSITORY`.
+
+  ```
+  DOCKER_REPOSITORY=docker.io
+  DOCKER_ORGANIZATION=splatform
+  ```
+
+  If the repository requires authentication the user has logged into it.
+
+  ```
+  docker login ...
+  ```
+
+## About the cluster to deploy the chart on.
+
+* The user has a functional CAP cluster, running all the necessary CF
+  and UAA roles/pods.
+
+## About the database to talk to
+
+The users knows the publicly reachable name of the host the database
+to talk to lives on. The user further knows the port on that host the
+database listens on, and the credentials (user and password) needed
+for the database to accept connections on the port. All this
+information is stored in the environment variables `DBHOST`, `DBPORT`,
+`DBUSER`, and `DBPASSWORD`.
+
+```
+DBHOST=mysql-host.in.some.domain
+DBPORT=3306
+DBUSER=root
+DBPASS=the-mysql-password
+```
+
+# Deploying the chart in three steps
+
+Please review the previous section and ensure that all the assumptions
+are satisfied.
+
+## CAP location and namespaces
+
+Determine the namespaces used for the CF and UAA roles in his CAP
```

Comment:
Up to now, the user was genderless (not sure if that was intentional). Yes, this is a silly nit and I wouldn't block merging for it :D

Reply:
Subconscious semi-intentional I believe. Will fix.

Reply:
Addressed with commit b951c6adc7d32f03d6f9a6ed411d7690ded72b7d